.devcontainer/dotnet/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "DotNet",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
```

Comment:
Why are you using node image in dotnet?

Reply:
Changed image to "mcr.microsoft.com/devcontainers/dotnet:7.0"